query/src/main/java/io/keen/client/java/RefreshRate.java

```diff
@@ -0,0 +1,36 @@
+package io.keen.client.java;
+
+import java.util.Locale;
+
+/**
+ * Helpers and constants for Saved/Cached Query refresh rate.
+ *
+ * @author masojus
+ */
+public final class RefreshRate {
+    private RefreshRate() {}
+
+    // The refresh rate range empirically is [14400, 86400] seconds, inclusive at both boundaries.
+    // A refresh rate of 0 means caching is turned off.
+    // TODO : Docs on the website are wrong as of 3/14/17, as has been reported. Get that fixed.
```

Comment:
Yeah this TODO will go away no matter what before merging. It's just so you and other reviewers see it and so it's in the git history, because it needs to be fixed but is external to the SDK.